src/p11_child/p11_child_openssl.c

```diff
@@ -1716,9 +1716,14 @@ errno_t do_card(TALLOC_CTX *mem_ctx, struct p11_ctx *p11_ctx,
                     goto done;
                 }
                 DEBUG(SSSDBG_TRACE_ALL,
-                      "Description [%s] Manufacturer [%s] flags [%lu] "
+                      "Description [%.*s] Manufacturer [%.*s] flags [%lu] "
                       "removable [%s] token present [%s].\n",
-                      info.slotDescription, info.manufacturerID, info.flags,
+                      p11_kit_space_strlen(info.slotDescription,
```

Comment:
Hi,

can you cast the return value of p11_kit_space_strlen() to int? See warnings, e.g. inhttps://s3.eu-central-1.amazonaws.com/sssd-ci/PR-5760/9/fedora34/build.log.

bye,
Sumit

Reply:
Thanks for noticing this. I have added the casts.